### ISSUE TITLE
Missing NVDA controller dll file after exporting project

### ISSUE THREAD

**Issue by btzr-io** [nightblade9/godot-nvda-integration#1]
Is there any guide or info on how should I export my game with this integration ?

During the jam, I had some issues after exporting my project, until I manually copy the dlls files into the exporting directory.
Is there a way to configure this so is included by default ?

**Comment by nightblade9** [comment]
Hey, sorry about that! I didn't end up using this, so I have no idea. I suspect you're right: you need the .DLLs present when you export the game.

As far as I know, there is no way to do this manually; you have to update your export config to include `*.dll` or the DLL names specifically.

If you end up figuring this out, could you open a PR and update the README?
